File: RestServer/WebApp/SocketClient/Model/BackLogItemEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.SocketClient.Model
{
    public class BacklogItemEntity
    {
        public string task;
        public string points;
        public string priority;

        public void SetValues(string task, string points, string priority)
        {
            this.task = task;
            this.points = points;
            this.priority = priority;

        }

    }



}

File: src/Model/BacklogItemEntity.java
package Model;

public class BacklogItemEntity {

    public String task;
    public String points;
    public String priority;
    public int Id;

    public BacklogItemEntity(String task, String points, String priority) {
        this.task = task;
        this.points = points;
        this.priority = priority;
    }
}
